src/editions/EditionFactory.sol

```diff
@@ -228,15 +206,28 @@ contract EditionFactory is IBatchEditionMinter {
             string[] memory propertyValues = new string[](1);
             propertyValues[0] = data.creatorName;
 
-            ISingleBatchEdition(edition).setStringProperties(propertyNames, propertyValues);
+            edition.setStringProperties(propertyNames, propertyValues);
         }
 
+        // set the external url
         string memory externalUrl = data.externalUrl;
         if (bytes(externalUrl).length > 0) {
-            ISingleBatchEdition(edition).setExternalUrl(data.externalUrl);
+            edition.setExternalUrl(data.externalUrl);
+        }
+
+        // configure the minter
+        address minterAddr = data.minterAddr;
+        if (minterAddr != address(0)) {
+            edition.setApprovedMinter(minterAddr, true);
+        }
+
+        // configure the operator filter
+        bool enableDefaultOperatorFilter = data.enableDefaultOperatorFilter;
+        if (enableDefaultOperatorFilter) {
+            edition.enableDefaultOperatorFilter();
         }
```

Comment:
could you update the comment here to mention this is for opensea stuff? i racked my brain for a second trying to remember what this meant :sweat_smile:

Reply:
yes! Thinking about it some more, I think it would be more future proof to pass an `operatorFilter` address (optionally `address(0)` to disable) instead of a bool, so you get to pick the filter if the default OpenSea stuff ever changes

```
    function setOperatorFilter(address operatorFilter) public override onlyOwner {
        _setOperatorFilter(operatorFilter);
    }

    function enableDefaultOperatorFilter() public override onlyOwner {
        _setOperatorFilter(CANONICAL_OPENSEA_SUBSCRIPTION);
    }
```

Reply:
that's a better idea imo

Reply:
in 0468884